examples/tensorflow/classification/main.py

```diff
@@ -229,6 +229,9 @@ def run(config):
     }
 
     if 'train' in config.mode:
+        statistics = compression_ctrl.statistics()
+        logger.info(statistics.to_str())
+
```

Comment:
Could you log statistics using Compression_callbacks for the training step? Explicitly logging statistics before evaluation is good for me and doesn't require changes in this PR.